### DISCUSSION TITLE
Add repository list plugin

### DISCUSSION THREAD

**Discussion by Nixinova** [lowlighter/metrics#320] [Ideas]
It would be useful to add a repository list plugin to show a summary of all your repositories (with maybe a configurable limit). Like the repository template but appended to the main image.

Something like what you see in your repos list but without the tags and with some added Metrics info

![image](https://user-images.githubusercontent.com/42429413/119240551-69dc2d80-bba4-11eb-8231-6702d5f09c83.png)

Here's a mockup:

![image](https://user-images.githubusercontent.com/42429413/119240720-b5430b80-bba5-11eb-85d2-edba596dcf54.png)

So, for instance:

<img width=500px src=https://user-images.githubusercontent.com/42429413/119240733-d146ad00-bba5-11eb-8148-dfc47c7268e2.png>
<img width=500px src=https://user-images.githubusercontent.com/42429413/119240743-f20f0280-bba5-11eb-8b58-bf799b7a745f.png>

**Comment by lowlighter** [comment]
I really like the idea!

It shouldn't be too hard to gather the data for it (it's actually possible that most of repositories data is already available since these are used in `base` for repositories section) so most of the work would be towards the templating.

A configurable limit for users with lot of repositories seems nice.

Another option to choose to display only repositories with a minimum amount of stats/forks using a github query like filter (what is used in `notable` plugin) could be a nice addition too

**Reply by lowlighter** [reply]
I can start working on it, unless if you wish to implement it yourself and submit a PR for it 👍

**Reply by Nixinova** [reply]
Created a skeleton here https://github.com/Nixinova/fork_metrics/commit/1d75b9f30fd889209c224052ec49dad2ddb5797a but can't quite get it to load the partial.

Here's what I've got so far
![image](https://user-images.githubusercontent.com/42429413/120085074-504d5f80-c129-11eb-82e3-ff5407a6f111.png)

**Reply by lowlighter** [reply]
Nice 👍 
To load the templates you need to add it somewhere to [source/templates/classic/partials/_.json](https://github.com/Nixinova/fork_metrics/blob/main/source/templates/classic/partials/_.json) (the array define the order, so choose where you think it's more appropriate, the `config_order` will let users override it)
